src/map/utils/petutils.h

```diff
@@ -144,6 +145,7 @@ struct Pet_t
     , HPscale(0.f)
     , MPscale(0.f)
     , cmbDelay(0)
+    , m_dmgType(DAMAGE_TYPE::NONE)
```

Comment:
CI is complaining that this needs to come before `cmbDelay(0)`

Reply:
> CI is complaining that this needs to come before `cmbDelay(0)`

Thanks should be fixed now.